tv_apps/smart-system/js/browser_context_menu.js

```diff
@@ -204,14 +206,37 @@
     // context menu api
     if (detail.contextmenu && detail.contextmenu.items.length) {
       var that = this;
-      detail.contextmenu.items.forEach(function(choice, index) {
+      detail.contextmenu.items.forEach(function(choice) {
+        var label;
+        var clickAction;
+
+        if (AppInstallManager.isMarketplaceAppActive()) {
```

Comment:
- Put this logic to AppInstallManager via CustomEvent.
- Stop handling contextmenu here.